shapely/geos.py

```diff
@@ -834,6 +834,7 @@ def parallel_offset(geom, distance, resolution=16, join_style=1,
         self.methods['snap'] = self.GEOSSnap
         self.methods['shared_paths'] = self.GEOSSharedPaths
         self.methods['buffer_with_params'] = self.GEOSBufferWithParams
+        self.methods['covered_by'] = self.GEOSCoveredBy
```

Comment:
I can't find the other link to the changelog I looked at but it seems CoveredBy was added in GEOS 3.3.0 https://git.osgeo.org/gitea/geos/geos/src/tag/3.3.0/ChangeLog#L952 so I'm adding it here to LGEOS330.

Reply:
That's correct: https://git.osgeo.org/gitea/geos/geos/src/branch/master/NEWS#L380